Breadcrumbs.py

```diff
@@ -32,80 +38,169 @@ def get_row_indentation(points, view, tab_size, limit=1e20):
 def is_white_row(view, points):
   return all(view.substr(pt).isspace() for pt in reversed(points))
 
-def get_breadcrumb(view, points, limit):
+def get_breadcrumb(view, points, regex, separator):
```

Comment:
Good points. I could do some minor refactoring here and let you review it? As long as the project is just a few hundred lines it doesn't have to be super elegant.